src/camera.c

```diff
@@ -342,17 +342,8 @@ void func_8001CA10(Camera* camera) {
 }
 
 void func_8001CA24(Player* player, f32 arg1) {
```

Comment:
How do you feel about passing the camera into this function instead?

Just curious. 2 functions up has camera.

Reply:
Is this question for me or coco? If for me I'm fine with how it is. It's also similiar to `void func_8001F394(Player* player, f32* arg1)` with the playerIndex variable doing pointer magic for the camera lookup there.